packages/lightning-lsp-common/src/shared.ts

```diff
@@ -73,25 +73,44 @@ export function detectWorkspaceHelper(root: string): WorkspaceType {
         return WorkspaceType.CORE_PARTIAL;
     }
 
+    if (fs.existsSync(path.join(root, 'lwc.config.js'))) {
+        return WorkspaceType.STANDARD_LWC;
+    }
+
     const packageJson = path.join(root, 'package.json');
     if (fs.existsSync(packageJson)) {
         try {
-            // Check if package.json contains @lwc/engine
             const packageInfo = JSON.parse(fs.readFileSync(packageJson, 'utf-8'));
             const dependencies = Object.keys(packageInfo.dependencies || {});
-            if (dependencies.includes('@lwc/engine')) {
+            const devDependencies = Object.keys(packageInfo.devDependencies || {});
+            const allDependencies = [...dependencies, ...devDependencies];
+            const hasLWCdependencies = allDependencies.some(key => key.includes('@lwc/'));
```

Comment:
```suggestion
            const hasLWCdependencies = allDependencies.some(key => key.startsWith('@lwc/'));
```
Maybe we could use `startsWith`? Since we know that the pattern is in the beginning

Reply:
It should also just check for the `lwc` package. 

https://www.npmjs.com/package/lwc

Reply:
Hey @muenzpraeger the `lwc` packaged is checked on line 95.